vendor.mod

```diff
@@ -92,35 +95,41 @@ require (
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
 	github.com/tonistiigi/go-archvariant v1.0.0
-	github.com/vbatts/tar-split v0.11.5
+	github.com/vbatts/tar-split v0.11.6
 	github.com/vishvananda/netlink v1.3.1-0.20240922070040-084abd93d350
 	github.com/vishvananda/netns v0.0.4
 	go.etcd.io/bbolt v1.3.11
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.53.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.53.0
-	go.opentelemetry.io/otel v1.28.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.28.0
-	go.opentelemetry.io/otel/metric v1.28.0
-	go.opentelemetry.io/otel/sdk v1.28.0
-	go.opentelemetry.io/otel/trace v1.28.0
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.56.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.56.0
+	go.opentelemetry.io/otel v1.31.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.31.0
+	go.opentelemetry.io/otel/metric v1.31.0
+	go.opentelemetry.io/otel/sdk v1.31.0
+	go.opentelemetry.io/otel/trace v1.31.0
 	golang.org/x/mod v0.21.0
-	golang.org/x/net v0.29.0
-	golang.org/x/sync v0.8.0
+	golang.org/x/net v0.30.0
+	golang.org/x/sync v0.9.0
 	golang.org/x/sys v0.26.0
-	golang.org/x/text v0.18.0
+	golang.org/x/text v0.19.0
 	golang.org/x/time v0.6.0
-	google.golang.org/genproto/googleapis/api v0.0.0-20240701130421-f6361c86f094
-	google.golang.org/grpc v1.66.2
+	google.golang.org/genproto/googleapis/api v0.0.0-20241007155032-5fefd90f89a9
+	google.golang.org/grpc v1.68.0
 	google.golang.org/protobuf v1.35.1
 	gotest.tools/v3 v3.5.1
 	resenje.org/singleflight v0.4.1
 	tags.cncf.io/container-device-interface v0.8.0
 )
 
+require (
+	github.com/containerd/containerd v1.7.23 // indirect
```

Comment:
of course I was curious what's bringing in v1.7.23;

```bash
go mod graph | grep ' github.com/containerd/containerd@'
github.com/docker/docker github.com/containerd/containerd@v1.7.23
github.com/Microsoft/hcsshim@v0.12.9 github.com/containerd/containerd@v1.7.23
github.com/containerd/nydus-snapshotter@v0.14.0 github.com/containerd/containerd@v1.7.20
github.com/moby/buildkit@v0.17.1 github.com/containerd/containerd@v1.7.23
```

And it's probably the circular dependency that's making it pop up;

```bash
go mod graph | grep ' github.com/moby/buildkit@v0.17.1'
github.com/docker/docker github.com/moby/buildkit@v0.17.1
```

Reply:
Ah, yes, I was looking at the nydus dependency at some point, and looking if we can remove the dependency altogether for our vendoring; this was some first attempt at making the compressions more modular, but I didn't go beyond those steps (move them to separate packages);

- https://github.com/moby/buildkit/pull/5354

Reply:
For nydus we are just waiting for a tag but can bump to a digest

Reply:
Confirmed updating nydus will drop containerd 1.7 from vendor